tests/app/post.go

```diff
@@ -30,12 +31,17 @@ func NewPostHandler(options PostHandlerOptions) (sdk.PostHandler, error) {
 		return nil, errorsmod.Wrap(sdkerrors.ErrLogic, "feemarket keeper is required for post builder")
 	}
 
+	if options.ConsensusParamsKeeper == nil {
```

Comment:
Were we not able to get `ConsensusParams` from the sdk.Context?

Reply:
This is a good point.  Would be a big simplification.  Idk if there is a difference between the values.  from the context and the keeper

Reply:
See `getContextForTx` in `baseapp.go`, TLDR, it reads the value from state on tx execution. One edge-case tho would be if `ConsensusParams` are modified in the tx that we are calculating fees for tho?

Reply:
Ya looking at this now, the only way to update is from comet or from a gov-proposal (which would be executed in Begin/EndBlockers), so this shld be safu

Reply:
I'd prefer to use ctx.ConsensusParams, but if that's a big lift I'm down to backlog and keep things as they are now, LMK what is preferred

Reply:
Agreed. Will be cleaner and simpler. I can make the change now!